integrations/paddleocr/pyproject.toml

```
@@ -0,0 +1,161 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
+
+[project]
+name = "paddleocr-haystack"
+dynamic = ["version"]
+description = 'An integration of PaddleOCR with Haystack'
+readme = "README.md"
+requires-python = ">=3.9"
+license = "Apache-2.0"
+keywords = []
+authors = [
+  { name = "deepset GmbH", email = "info@deepset.ai" },
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = [
+  "haystack-ai>=2.19.0",
+  "paddleocr>=3.3.2",
+  "paddlex[serving]>=3.3.10",
+  "requests>=2.25.0",
+  "typing-extensions",
+]
+
+[project.urls]
+Documentation = "https://github.com/haystack-core-integrations/tree/main/integrations/paddleocr#readme"
+Issues = "https://github.com/haystack-core-integrations/paddleocr/issues"
+Source = "https://github.com/haystack-core-integrations/tree/main/integrations/paddleocr"
+
+[tool.hatch.build.targets.wheel]
+packages = ["src/haystack_integrations"]
+
+[tool.hatch.version]
+source = "vcs"
+tag-pattern = 'integrations\/paddleocr-v(?P<version>.*)'
+
+[tool.hatch.version.raw-options]
+root = "../.."
+git_describe_command = 'git describe --tags --match="integrations/paddleocr-v[0-9]*"'
+
+[tool.hatch.envs.default]
+installer = "uv"
+dependencies = ["haystack-pydoc-tools", "ruff"]
+
+[tool.hatch.envs.default.scripts]
+docs = ["pydoc-markdown pydoc/config.yml"]
```

```suggestion
docs = ["pydoc-markdown pydoc/config_docusaurus.yml"]
```
We recently changed this command in all integrations.